weaviate/data/crud_data.py

```diff
@@ -76,6 +77,8 @@ def create(
              - a class does not have a vectorization module.
              - The given vector was generated using the _identical_ vectorization module that is configured for the
              class. In this case this vector takes precedence.
+        tenant_key: str on None, optional
+            Defined in multitenancy config tenantKey setting.
 
             Supported types are `list`, 'numpy.ndarray`, `torch.Tensor` and `tf.Tensor`,
```

Comment:
line 83-84 are supposed to be under `vector`, so the correct way would be this:
```
        vector: Sequence or None, optional
            Embedding for the object.
            Can be used when:
             - a class does not have a vectorization module.
             - The given vector was generated using the _identical_ vectorization module that is configured for the
             class. In this case this vector takes precedence.
            Supported types are `list`, 'numpy.ndarray`, `torch.Tensor` and `tf.Tensor`,
            by default None.
        tenant_key: str on None, optional
            Defined in multitenancy config tenantKey setting.
```

Reply:
Thanks! I have fixed that